profiler/src/ProfilerEngine/Datadog.Profiler.Native/cgroup.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
```

Comment:
Don't forget the header
```
// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2022 Datadog, Inc.
```